apps/researcher/src/app/[locale]/error.tsx

```diff
@@ -0,0 +1,15 @@
+'use client'; // Error components must be Client components
+
+import {useEffect} from 'react';
+
+export default function Error({error}: {error: Error}) {
+  useEffect(() => {
+    console.error(error);
+  }, [error]);
+
+  return (
+    <div data-testid="error">
+      <h2>Something went wrong!</h2>
```

Comment:
Is there a way to localize this message? (Also applies to our existing `error.tsx` files)

Reply:
I have removed the old error file. Because this will never load. In the `[locale]` level it is possible to add translations. So I will add it here.